packages/dds/tree/src/test/shared-tree/summaryBenchmarkSizes.ts

```diff
@@ -0,0 +1,173 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
+import { IsoBuffer } from "@fluidframework/common-utils";
+import { makeRandom } from "@fluid-internal/stochastic-test-utils";
+import { TransactionResult } from "../../checkout";
+import { FieldKinds, singleTextCursor } from "../../feature-libraries";
+import { ISharedTree } from "../../shared-tree";
+import { FieldKey, rootFieldKey, rootFieldKeySymbol, TreeValue } from "../../tree";
+import { brand } from "../../util";
+import { TestTreeProvider } from "../utils";
+import { fieldSchema, GlobalFieldKey, namedTreeSchema, SchemaData } from "../../schema-stored";
+// eslint-disable-next-line import/no-internal-modules
+import { PlacePath } from "../../feature-libraries/sequence-change-family";
+
+const globalFieldKey: GlobalFieldKey = brand("globalFieldKey");
+
+describe("Summary size benchmark", () => {
+    it("with no nodes.", async () => {
+        const provider = await TestTreeProvider.create(1);
+        const tree = provider.trees[0];
+
+        const { summary } = tree.getAttachSummary();
+        const summaryString = JSON.stringify(summary);
+        const summarySize = IsoBuffer.from(summaryString).byteLength;
+        assert(summarySize < 1000);
+    });
+    it("with 1 inserted node.", async () => {
```

Comment:
I don't get why all these tests are about "inserted nodes".

All nodes in trees were inserted at some point and what should matter in the summary size is the shape of the tree when summarized, not how the nodes got there.

Reply:
Changed phrasing to "for a wide/narrow tree with x nodes"